package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "measure-text",
+  "version": "1.0.0",
```

Comment:
Let's start at `0.0.1` until we've gotten some better documentation down, and decided on infrastructure support.